tests/unit/chronology/tests.py

```diff
@@ -44,3 +56,121 @@ def test_str(model, kwargs, expected):
 
 def test_default_json_field():
     assert models.default_json_field() == {}
+
+
+@pytest.mark.django_db
+def test_festival_draft():
+    festival = FestivalFactory(
+        end_proposal=None,
+        end_time=None,
+        start_proposal=None,
+        start_publication=None,
+        start_time=None,
+    )
+    assert festival.status == Festival.DRAFT
+
+
+def _dt(hours):
```

Comment:
I  think I see it for 3rd time. I do not know if there's a place where we could keep it...